articles/virtual-machines/windows/multiple-nics.md

```diff
@@ -183,6 +183,13 @@ $vm.NetworkProfile.NetworkInterfaces
 Remove-AzureRmNetworkInterface -Name "myNic3" -ResourceGroupName "myResourceGroup" | `
     Update-AzureRmVm -ResourceGroupName "myResourceGroup"
 ```
+^^^^ THIS IS THE WRONG CMDLET AND THE WRONG TYPE OF DATA FOR "-Name" ANYWAY  ^^^^

```

Comment:
And, remove 183-184 which contain the incorrect command layout. Just leave in what the command should like.